src/java/org/apache/cassandra/config/AccordSpec.java

```diff
@@ -204,6 +205,14 @@ public MinEpochRetrySpec()
         }
     }
 
+    public static class FetchRetrySpec extends RetrySpec
```

Comment:
`RetrySpec` has the following constructor

```
public RetrySpec(MaxAttempt maxAttempts, LongMillisecondsBound baseSleepTime, LongMillisecondsBound maxSleepTime)
```

Nothing stops us from adding

```
public RetrySpec(MaxAttempt maxAttempts)
```

or a `create` method... I think i did things this way mostly because of repair but nothing is wrong with `RetrySpec fetchRetry = RetrySpec.create(100);` IMO

I am 100% fine with this class (assuming the ref test is fixed), I am also fine with creating new methods in `RetrySpec` to get the same behavior